backup-restore/recipes/getRecipeWorkflow.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/zsh
```

Comment:
Linux by default does not have zsh. Change to bash. That will also be consistent other scripts in the repo.

Reply:
Modified.